### PR TITLE
Remove landscape failures

### PR DESCRIPTION
**Pull request by syre**
PR for removing some of the code health failures from Landscape:
- use new-style classes
- specify string format arguments as logging function parameters
- remove deprecated `load_stylesheet_pyqt5` and replace with `load_stylesheet`

### CODE REVIEW

File: pynocchio/comic_file_loader_tar.py [hunk 1]
```diff
@@ -71,20 +71,18 @@ def load(self, filename):
...
-                    logger.info('Adding page %s' % name)
+                    logger.info('Adding page %s', name)
                     try:
                         data = tar.read(name)
                         self.data.append(Page(data, name, page))
                         page += 1
                     except tarfile.ExtractError as exc:
-                        logger.exception('Error in extract %s file. %s' %
-                                         (name, exc))
+                        logger.exception('Error in extract %s file. %s', name, exc)
```
Comment (mstuttgart) [inline pynocchio/comic_file_loader_tar.py]: @syre please change to 

```python
logger.exception('Error in extract ', name, ' file. ', exc)
```

File: pynocchio/comic_file_loader_tar.py [hunk 2]
```diff
@@ -71,20 +71,18 @@ def load(self, filename):
...
                     except tarfile.ReadError as exc:
-                        logger.exception('Error in read %s file. %s' % (name,
-                                                                        exc))
+                        logger.exception('Error in read %s file. %s', name, exc)
```
Comment (mstuttgart) [inline pynocchio/comic_file_loader_tar.py]: @syre please change to 

```python
logger.exception('Error in read ', name, ' file. ', exc)
```

File: pynocchio/comic_file_loader_zip.py [hunk 3]
```diff
@@ -53,16 +53,15 @@ def load(self, filename):
...
                 if get_file_extension(name).lower() in IMAGE_FILE_FORMATS:
-                    logger.info('Adding page %s' % name)
+                    logger.info('Adding page %s', name)
                     try:
                         self.data.append(Page(zf.read(name), name, page))
                         page += 1
                     except zipfile.BadZipfile as exc:
-                        logger.exception('Error in read %s file. %s' % (name,
-                                                                        exc))
+                        logger.exception('Error in read %s file. %s', name, exc)
```
Comment (mstuttgart) [inline pynocchio/comic_file_loader_zip.py]: @syre please change to 

```python
logger.exception('Error in read ', name, ' file. ', exc)
```

File: pynocchio/main_window_model.py [hunk 4]
```diff
@@ -56,7 +56,7 @@ def load_current_directory(self):
         return self.settings_manager.load_current_directory()
 
     def load(self, filename, initial_page=0):
-        logger.info('Loading %s at %i' % (filename, initial_page))
+        logger.info('Loading %s at %i', filename, initial_page)
```
Comment (mstuttgart) [inline pynocchio/main_window_model.py]: @syre please change to 

```python
logger.exception('Loading  ', filename, ' at ', initial_page)
```

**Comment by syre** [comment]
@mstuttgart i believe the logging call are correct as this is the way to do string substitution in logging (as argument) https://docs.python.org/3/library/logging.html#logging.Logger.debug

**Comment by mstuttgart** [comment]
Sorry @syre . You're right: https://fangpenlin.com/posts/2012/08/26/good-logging-practice-in-python/

**Comment by mstuttgart** [comment]
Greate work @syre . Please fix the flake8 errors: https://travis-ci.org/pynocchio/pynocchio/jobs/439123980#L743

**Comment by syre** [comment]
Thanks @mstuttgart i have done so now

**Pull request merged**